pkg/utils/helpers.go

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import (
+	fluxhelmv2beta1 "github.com/fluxcd/helm-controller/api/v2beta1"
+	"sigs.k8s.io/yaml"
+	"strings"
+)
+
+func ConvertToDNS1123(in string) string {
+	return strings.ReplaceAll(in, "_", "-")
+}
+
+func ToInitials(in string) (out string) {
```

Comment:
Not relevant to this PR, but I have come to the conclusion that this is probably not the right way to do it - we need to generate a unique prefix instead of using the initials when pushing charts to the staging registry

Reply:
💯